.travis/format.sh

```diff
@@ -84,6 +84,9 @@ else
     format_changed
 fi
 
+# Lint Cython files (see .flake8 in the root directory)
+flake8
```

Comment:
How long does this take to run?

Reply:
I changed this now so only files that were modified are run through flake8, so the runtime is now similar to what it was before (~20 sec).